### DISCUSSION TITLE
Release API not Providing Latest Release

### DISCUSSION THREAD

**Discussion by Masterain98** [community/community#77149] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

I have a script used to collect the latest three releases of my repository via releases REST API (https://api.github.com/repos/DGP-Studio/Snap.Hutao/releases?per_page=30&page=0). 

It used to be good, but today after I make the latest release [1.8.1](https://github.com/DGP-Studio/Snap.Hutao/releases/tag/1.8.1), it never shows up in the releases list API described above, even it is showing in the latest release API (https://api.github.com/repos/DGP-Studio/Snap.Hutao/releases/latest). 

Could anyone please explain to me what's wrong with it?

**Comment by Masterain98** [comment, marked as answer]
Oh, I see it at the bottom of the first page. Seems like there was a cache blocking the latest data.

**Comment by leonardowf** [comment]
Thanks for opening the discussion.
I also observed the same behavior in one of my repos.
Making a release the latest release made it not appear anymore on the response of the releases endpoint.
I will modify my script to pick the latest instead
